pkg/log/log.go

```diff
@@ -39,7 +40,8 @@ func init() {
 		logLevel = logrus.InfoLevel
 	}
 	log.SetLevel(logLevel)
-	log.Formatter = &logrus.TextFormatter{ForceColors: true, FullTimestamp: true}
+	forceColors, _ := strconv.ParseBool(os.Getenv("LOG_FORCE_COLORS"))
```

Comment:
what value do we get when `ParseBool` fails? We should check the error and set forceColors=false if is not nil

Reply:
ParseBool returns `false` as default value. (Even in case of an error)

Reply:
I still think it's better to handle error for readability